packages/jest-runner/corpus.ts

```diff
@@ -44,16 +54,47 @@ export class Corpus {
 
 	private createMissingDirectories() {
 		fs.mkdirSync(this._inputsDirectory, { recursive: true });
+		fs.mkdirSync(this._corpusDirectory, { recursive: true });
 	}
 }
 
-const buildInputsDirectory = (
+/**
+ * Build a directory path based on the test file path and the Jest path elements.
+ * @param testSourceFilePath - The path to the test file
+ * @param testJestPathElements - Jest path elements, e.g. ["describe text", "it text"]
+ * @param corpusOrInputs - If false, the inputs directory is built (default), otherwise the corpus directory
+ */
+const buildDirectory = (
 	testSourceFilePath: string,
-	testJestPathElements: string[]
+	testJestPathElements: string[],
+	corpusOrInputs = false
 ): string => {
-	const root = path.parse(testSourceFilePath);
-	const pathElements = testJestPathElements.map(replaceSpacesWithUnderscore);
-	return path.join(root.dir, root.name, ...pathElements) + path.sep;
+	const safeTestJestPathElements = testJestPathElements.map(
+		replaceSpacesWithUnderscore
+	);
+	const inputsRoot = path.parse(testSourceFilePath);
+	const testName = inputsRoot.name;
+	let mainDir = inputsRoot.dir;
+	if (corpusOrInputs) {
+		mainDir = path.join(
+			findDirectoryWithPackageJson(inputsRoot).dir,
+			".ci-corpus"
```

Comment:
In Jazzer the directory is called `.cifuzz-corpus`. We should use the same name as well. Extract that out in a constant for easier reference?